server/tests/audit_math/test_sampler.py

```diff
@@ -182,3 +183,39 @@ def test_ballot_labels():
         sample = sampler.draw_sample(SEED, manifest, 100, 0)
         for (_, (batch, ballot_number), _) in sample:
             assert 1 <= ballot_number <= max(manifest[batch])
+
+def test_correct_sample_size():
+    name = "Dekalb Primary 2022"
+
+    info_dict = {
+        "Dee Dawkins-Haigler": 6672,
+        "Bee Nguyen": 37882,
+        "ballots": 45743,
+        "numWinners": 1,
+        "votesAllowed": 1,
+    }
+
+    contest = Contest(name, info_dict)
+
+    seed = "84976858374874550815"
+    alpha = 0.05
+    sample_size = 6
+
+    batches = {}
+    with open("server/tests/audit_math/audit_data/dekalb_2022_primary_batch_totals.csv") as csv:
+        for row in DictReader(csv):
+            batches[("Dekalb", row["Batch Name"])] = {
+                name: {
+                    "Dee Dawkins-Haigler": int(row["Dee Dawkins-Haigler"]),
+                    "Bee Nguyen": int(row["Bee Nguyen"]),
+                }
+            }
+
+    with open("server/tests/audit_math/audit_data/dekalb_2022_primary_manifest.csv") as csv:
+        for row in DictReader(csv):
+            batches[("Dekalb", row["Batch Name"])][name]["ballots"] = int(row["Number of Ballots"])
+
+    sample = sampler.draw_ppeb_sample(seed, contest, sample_size, 0, batches)
+
+    assert len(sample) == sample_size
```

Comment:
Since `test_dekalb_primary_2022` includes all of what's here in `test_correct_sample_size` and more, we could maybe just get rid of this test in favor of `test_dekalb_primary_2022` - I don't feel strongly about this!

Reply:
yeah, you're probably right about that. I'll kill it!